Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'transproc', github: 'aflatter/transproc', branch: 'unwrap'
```

Comment:
I just released transproc 0.1.3 witch `unwrap` support so please remove this and update gemspec to specify transproc ~> 0.1, >= 0.1.3